app/models/result.rb

```diff
@@ -118,8 +118,8 @@ def check_for_nil_marks
     if marks.where(mark: nil).first &&
           marking_state == Result::MARKING_STATES[:complete]
       errors.add(:base, I18n.t('common.criterion_incomplete_error'))
-      false
+      return false
     end
-    true
+    return true
```

Comment:
@peterzhao333 So Hound is our Ruby style-checker. It's saying that this change was not necessary, and doesn't conform with the Ruby style guidelines.